source/documentation/runbooks/lpa.html.rb.erb

```diff
@@ -0,0 +1,90 @@
+
+### Incident response hours
+
+    - 9am to 5pm Monday to Friday, excluding bank holidays, public holidays.
+
+For notifications recived outside this window we have no obligation to respond until the hours above.
+
+### Impact of an outage
+
+Users will not be able to create, access or download their LPA documents.
+
+### Incident contact details
+
+Slack: [#opg-lpa-live](https://mojdt.slack.com/archives/C01BDM8T67J)
+
+email: [opg-lpa-live@digital.justice.gov.uk](mailto:opg-lpa-live@digital.justice.gov.uk)
+
+## URLs
+
+### Service URLs:
+
+[https://www.lastingpowerofattorney.service.gov.uk/](https://www.lastingpowerofattorney.service.gov.uk/)
+
+[https://admin.lpa.opg.service.justice.gov.uk/](https://admin.lpa.opg.service.justice.gov.uk/)
+
+### Other URLs
+
+|Name                           |Address                                                                                |
+|-------------------------------|---------------------------------------------------------------------------------------|
+| Healthcheck ELB               |     https://www.lastingpowerofattorney.service.gov.uk/ping/elb                        |
+| Healthcheck Availability      |     https://www.lastingpowerofattorney.service.gov.uk/ping/                           |
+| Healthcheck Pingdom           |     https://www.lastingpowerofattorney.service.gov.uk/ping/pingdom                    |
+| Healthcheck Json              |     https://www.lastingpowerofattorney.service.gov.uk/ping/json                       |
+| GitHub Repository             |     https://github.com/ministryofjustice/opg-lpa                                      |
+
+## Hosting environment
+- AWS - We can log an emergency call with [aws support](https://console.aws.amazon.com/support/home#/) and will get an immediate call back.
+- AWS Account IAM Roles can be found [here](https://ministryofjustice.github.io/opg-new-starter/amazon.html#lpa)
+
+## Service team contact
+
+[opg-lpa-live@digital.justice.gov.uk](opg-lpa-live@digital.justice.gov.uk)
+
+## Other Information
+
+### Releases
+
+We operate a CI/CD model, releases will be continuously deployed as and when required.
+
+### Automatic alerts
+
+| Alert               | Source     | Application     | Description                                                                                |
+|---------------------|------------|-----------------|--------------------------------------------------------------------------------------------|
+| 5xx errors in app   | Cloudwatch | Front           | Alerts on when we have a status code of 5xx appear in the application logs                 |
+| 5xx from ALB        | Cloudwatch | Front           | Alerts where the application load balancer receives a 5XX response                         |
+
+### Consumers of this service
+
+N/A
+
+### Services consumed by this
+
+- SendGrid
+- Sirius LPA
+
+### Restrictions on access
+
+None for production, other environments and admin restricted to users on VPN, DOM1, OPG Wifi
```

Comment:
Maybe break this down by service front end to avoid any confusion?

Reply:
```suggestion
public front: no restrictions for production, other environments are restricted to VPN, DOM1 and OPG Wifi 
admin front: restricted to users on VPN, DOM1, OPG Wifi on all environments
```

Reply:
@mattmachell - does this look ok?

Reply:
Looks good.